packages/SkeletonProvider/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
This file shouldn't be committed.

Reply:
Currently in community, the lock files will continually appear due to unmatching dependencies. For now I think we're better off just committing these.